imports/plugins/core/taxes/client/settings/custom.html

```diff
@@ -49,6 +49,7 @@
         resetOnSuccess=true
       }}
         <div>
+          {{> afFieldInput name='shopId' value=shopId type='hidden' }}
```

Comment:
Does adding this serves a purpose? We are not giving option to set it, right?

Reply:
It's a hidden field, so it will be added to the form submit. That's the main fix.